pkg/usagestats/stats.go

```diff
@@ -168,13 +168,29 @@ func NewString(name string) *expvar.String {
 	return expvar.NewString(statsPrefix + name)
 }
 
-// Target sets the target name.
+// Target sets the target name. This can be set multiple times.
 func Target(target string) {
+	existing := expvar.Get(statsPrefix + targetKey)
+	if existing != nil {
```

Comment:
Can we apply this to all expvar above: NewFloat, NewInt, NewString, Statistics, Counter  .....